src/app/plant.ts

```diff
@@ -0,0 +1,7 @@
+export interface Plant {
+  id: number;
+  name: string;
+  plantType: string;
+  imgUrl: string;
+  plantImgs: string[];
```

Comment:
Great way of making images dynamic! Looks like we can get rid of imgUrl and just use the plantImgs array.